.github/workflows/ruby.yml

```
@@ -12,9 +12,10 @@ jobs:
   build:
     runs-on: ubuntu-latest
 
+    continue-on-error: true
     strategy:
       matrix:
-        ruby: [2.5.9, 2.6.10, 2.7.7]
+        ruby: [2.5.9, 2.6.10, 2.7.7, 3.0.5, 3.1.3]
         gemfile: [Gemfile, gemfiles/rails60.gemfile, gemfiles/rails52.gemfile, gemfiles/rails51.gemfile]
```

You can use `exclude` to remove bad combos from the matrix: 

```suggestion
        ruby: [2.5.9, 2.6.10, 2.7.7, 3.0.5, 3.1.3, 3.2.0]
        gemfile: [Gemfile, gemfiles/rails60.gemfile, gemfiles/rails52.gemfile, gemfiles/rails51.gemfile]
        exclude: 
          - ruby: 3.1.3
            gemfile: gemfiles/rails60.gemfile
          - ruby 3.2.0
            gemfile: gemfiles/rails60.gemfile
```

etc.

---

thanks for the tip! I'll try it out next time.